### PR TITLE
add unit test for group commands

### PR DESCRIPTION
**Pull request by karakayasemi**
## Motivation and Context

More code coverage
## How Has This Been Tested?

 SQLite and MySQL

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tests/Core/Command/Group/AddMemberTest.php [hunk 1]
```diff
@@ -0,0 +1,214 @@
+<?php
+/**
+ * @author Semih Serhat Karakaya <karakayasemi@itu.edu.tr>
+ *
+ * @copyright Copyright (c) 2017, ownCloud, Inc.
```
Comment (DeepDiver1975) [inline tests/Core/Command/Group/AddMemberTest.php]: GmbH

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]
as discussed on irc yesterday - please use the command tester - https://github.com/owncloud/core/blob/master/tests/Core/Command/Apps/AppsDisableTest.php

Thanks a lot!

**Comment by karakayasemi** [comment]
@DeepDiver1975  Command tester is really good to use. I rewrote it all of them by using the command tester.

File: tests/Core/Command/Group/AddMemberTest.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    private $commandTester;
+
+    protected function setUp() {
+        parent::setUp();
+
+        $command = new AddMember(\OC::$server->getGroupManager(), \OC::$server->getUserManager());
+        $this->commandTester = new CommandTester($command);
+
+        $user1 = \OC::$server->getUserManager()->createUser('user1', 'password');
+        \OC::$server->getUserManager()->createUser('user2', 'password');
```
Comment (DeepDiver1975) [inline tests/Core/Command/Group/AddMemberTest.php]: You can use the User trait - https://github.com/owncloud/core/blob/master/apps/dav/tests/unit/Command/CleanupChunksTest.php#L40
This gives you some short cuts like https://github.com/owncloud/core/blob/9fd116cedf00cf4520c72dd696dc0e02b9501660/apps/dav/tests/unit/Command/CleanupChunksTest.php#L71-L72

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

**Comment by karakayasemi** [comment]
Is there a problem in Jenkins? or something is wrong with PR

**Comment by DeepDiver1975** [comment]
Jenkins needs restart

**Comment by DeepDiver1975** [comment]
> Jenkins needs restart

job restarted

**Review by DeepDiver1975 — APPROVED** [review]

**Comment by DeepDiver1975** [comment]
Just rebased to make use of jobs running on drone

**Comment by DeepDiver1975** [comment]
@karakayasemi THX a lot! Can I ask you to backport this to stable10? THX

**Comment by DeepDiver1975** [comment]
backport open

**Pull request merged**